ansible/roles/test/tasks/check_testbed_interfaces.yml

```diff
@@ -0,0 +1,57 @@
+- block:
+  - name: Gathering lab graph facts about the device
+    conn_graph_facts: host={{ inventory_hostname }}
+    connection: local
+
+  - name: Fanout hostname
+    set_fact: fanout_switch={{ device_conn['Ethernet0']['peerdevice'] }}
+
+  - name: Check Fanout interfaces
+    local_action: shell ansible-playbook -i lab fanout.yml -l {{ fanout_switch }} --tags check_interfaces_status
+    ignore_errors: yes
+
+  when: 
+    - check_fanout is defined
+
+- block:
+  - name: Get Portchannel status 
+    shell: show interfaces portchannel
+    register: portchannel_status
+    ignore_errors: yes
+
+  - name: Get teamd dump
+    shell: teamdctl '{{ item }}' state dump
+    with_items: "{{ minigraph_portchannels }}"
+    ignore_errors: yes
+    when:
+      - minigraph_portchannels is defined
+
+  - name: Define testbed_name when not obtained
+    set_fact:
+      testbed_name: "{{ inventory_hostname + '-' + topo }}"
+    when: testbed_name is not defined
+
+  - name: Gathering testbed information
+    test_facts: testbed_name="{{ testbed_name }}"
+    connection: local
+    ignore_errors: yes
+
+  - name: Gather vm list from Testbed server
+    local_action: shell ansible-playbook testbed_vm_status.yml -i veos -l "{{ testbed_facts['server'] }}"
```

Comment:
where the output does go?

Reply:
All gathered info goes to the execution console, where it will be extracted if needed.

Reply:
Can we save the output into the variable and output it as
```
debug: var: output.stdout_lines
```

Reply:
Here is the same explanation as in the first conversation.

When _interfaces.yml_ fails,
it calls _check_testbed_interfaces.yml_ with _check_vms: true_

_check_testbed_interface.yml_ plays another playbook - _testbed_vm_status.yml_:
`local_action: shell ansible-playbook testbed_vm_status.yml -i veos -l "{{ testbed_facts['server'] }}"`

_testbed_vm_status.yml_ has register/debug of output 
```
- hosts: servers:&vm_host
  tasks:
  - name: Get VM statuses from Testbed server
    shell: virsh list
    register: virsh_list
  - name: Show VM statuses
    debug: msg="{{ virsh_list['stdout_lines'] }}"
```
but it only will be shown in verbose mode _-vvvvv_ of the outer playbook: